gym_ignition/__init__.py

```diff
@@ -32,8 +32,12 @@
 # GYMPP C++ ENVIRONMENTS
 # ======================
 
+import numpy as np
+inf = float(np.finfo(np.float32).max)
```

Comment:
This may be confusing, as `numpy.inf` exists ( https://www.numpy.org/devdocs/reference/constants.html#numpy.Inf ) and it has a different value from the maximum floating point number.

Reply:
Right, renamed to `max_float`